admins.py

```diff
@@ -20,6 +22,13 @@
     resp = requests.get(admins_with_1fa_url, headers=headers)
 
     bad_admins = resp.json()
-    print 'The following admins DO NOT HAVE 2FA:'
-    for a in bad_admins:
-        print a['login']
+
+    # for orgs that don't yet support the new API, the 'role' filter is
+    # not honored. Filter those out by checking the 'site_admin' key.
+    real_admins = [x for x in bad_admins if x['site_admin']]
+    if len(real_admins):
```

Comment:
improper upbringing? I tend to forget some of the truthy/falsey nuances, so go old school with integers.